Source/WebKit/UIProcess/ios/fullscreen/WKFullScreenViewController.mm

```diff
@@ -452,6 +452,14 @@ - (void)setPictureInPictureActive:(BOOL)active
     [_pipButton setSelected:active];
 }
 
+- (void)setInWindowFullscreenActive:(BOOL)active
+{
+    if (_inWindowFullscreenActive == active)
+        return;
```

Comment:
Nit: not sure this check buys anything

Reply:
Yeah, I actually just got rid of this change entirely.